src/components/mint-button-secondary/mint-button-secondary.scss

```diff
@@ -1,4 +1,4 @@
-@import '~sass/main.scss';
+@import '~sass/main';
```

Comment:
This thing breaks Travis integration (scss-lint).
There's an issue to fix that: #535